web/ui/react-app/yarn.lock

```diff
@@ -927,6 +927,13 @@
   dependencies:
     "@fortawesome/fontawesome-common-types" "^0.2.25"
 
+"@fortawesome/free-regular-svg-icons@^5.7.1":
```

Comment:
Sorry, didn't explicitly call it out earlier, but the yarn.lock entry can now be removed too :)